tests/Test/Compiler.hs

```diff
@@ -53,21 +52,21 @@ testIf handleResult filePaths =
   where
     setupTest filePath =
       do  source <- readFile filePath
-          let result = Compiler.compile "elm-lang" "core" source Map.empty
+          let result = Compiler.compile "elm-lang" "core" True source Map.empty
```

Comment:
Not sure if calling Compiler.compile with `True` for the `isRoot` parameter is correct here. Can you confirm/deny, @evancz?